Library/Homebrew/manpages/brew-cask.1.md

```diff
@@ -255,7 +252,7 @@ Environment variables specific to Homebrew-Cask:
     the command-line. This is particularly useful to make options persistent.
     For example, you might add to your .bash_profile or .zshenv something like:
 
-               export HOMEBREW_CASK_OPTS='--appdir=/Applications --caskroom=/etc/Caskroom'
+               export HOMEBREW_CASK_OPTS='--appdir=/Applications --fontdir=/Library/Fonts'
```

Comment:
Instead of `--appdir=/Applications`, it should be `--appdir=~/Applications`, since `/Applications` is now the default, you wouldn’t set a flag for it.